airflow/providers/ftp/hooks/ftp.py

```diff
@@ -20,7 +20,7 @@
 import datetime
 import ftplib
 import os.path
-from typing import List, Optional
+from typing import List, Optional, Any, Union
```

Comment:
```suggestion
from typing import List, Optional, Any
```

we can remove Union once https://github.com/apache/airflow/pull/11107/files#r493825856 is implemented